capstone/src/main/webapp/js/maps.js

```diff
@@ -0,0 +1,68 @@
+/* eslint no-unused-vars: ["error", {"varsIgnorePattern": "createMap"}] */
+
+/**
+ * Function that creates map markers and info windows.
+ */
+function createMap() {
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 39.8097343, lng: -98.5556199}, zoom: 5});
+
+  const mapsDataList = fetchData();
+
+  const infoWindow = new google.maps.InfoWindow();
+  for (const mapsData of mapsDataList) {
+    const latitudeLongitude = new google.maps.LatLng(
+        mapsData['latitude'],
+        mapsData['longitude']);
+
+    const marker = new google.maps.Marker({
+      position: latitudeLongitude,
+      map: map,
+      title: mapsData['name'],
+    });
+
+    (function(marker, mapsData) {
+      google.maps.event.addListener(marker, 'click',
+          function() {
```

Comment:
```suggestion
          () => {
```
Arrow functions are preferred according to the [style guide](https://google.github.io/styleguide/jsguide.html#features-classes-old-style).